### DISCUSSION TITLE
Cloned site to dev site. Dev site about to go live. How to reconnect Meeting Guide App

### DISCUSSION THREAD

**Discussion by onemooresam** [code4recovery/12-step-meeting-list#1285] [Q&A] [answered]
I cloned the live site to a dev site on a new host (Siteground). While in dev, I removed sharing to Meeting Guide App and GSSA (Georgia State Service Assembly). We're ready to make the dev site the live site. How do I re-add Meeting Guide App and GSSA to those authorized to access the restricted feed?

This seems the right order of operations:
1. On Siteground, I'll change the domain from the temp domain to atlantaaa.org.
2. Change DNS to point atlantaaa.org to new host IP
3. Now, with permanent URLs in place, reconnect Meeting Guide and GSSA to our restricted feed

**Comment by joshreisner** [comment]
that sounds right!

**Reply by onemooresam** [reply]
Thanks, Josh. Regarding "How do I re-add Meeting Guide App and GSSA to those authorized to access the restricted feed?" do I need to resubmit via the connection form at https://meetingguide.helpdocs.io/article/g0ykqkdq0u-connecting-to-meeting-guide-step-by-step? How do I re-add GSSA to the authorized apps? 

Meeting Guide and GSSA were listed. Now they're gone. How do I put them back?

**Reply by joshreisner** [reply, marked as answer]
i'd:

* go to Meetings > Settings 
* select "Restricted" under Feed Management if it's not already
* create an Authorized App link for Meeting Guide
* copy the URL of that link and test it here: https://meetingguide.org/validate
* if that works, resubmit the connection form and let them know about the new link
* create an Authorized App link for Area
* copy the URL of that link and send it to them

<img width="518" alt="Screenshot 2023-12-20 at 11 16 42 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/2b905f41-89de-4eee-a82f-37f89ef08a38">

**Reply by onemooresam** [reply]
Perfect, thank you! I wasn't clear that I could add any name and that would gen a URL that I submit to that entity.
